src/services/ledger/schema.ts

```diff
@@ -160,3 +160,20 @@ transactionSchema.index({
 })
 
 export const Transaction = mongoose.model("Medici_Transaction", transactionSchema)
+
+const transactionMetadataSchema = new Schema({
```

Comment:
missing `_ledgerTransactionId`?

Reply:
It looks like for the Schemas where the `_id` is mongo-generated we never define this property in the Schema definition.